### PR TITLE
WIP: Let notes reference schools/teams

### PR DESCRIPTION
**Pull request by sleidig**

### CODE REVIEW

File: src/app/child-dev-project/attendance/attendance.service.ts [hunk 1]
```diff
@@ -254,19 +264,34 @@ export class AttendanceService {
...
+    const schoolParticipants = await this.loadParticipantsOfGroups(
+      activity.linkedGroups
     );
-    const schoolParticipants = await Promise.all(childIdPromises);
     instance.date = date;
     instance.subject = activity.title;
     instance.children = [
       ...new Set(activity.participants.concat(...schoolParticipants)), //  remove duplicates
     ];
+    instance.schools = activity.linkedGroups;
```
Comment (TheSlimvReal) [inline src/app/child-dev-project/attendance/attendance.service.ts:275]: Maybe it was not to best idea to call it `linkedGroups` for now we still use the school term in all other cases and I think this makes is a little hard to follow

File: src/app/child-dev-project/schools/school-select/school-select.component.html [hunk 2]
```diff
@@ -3,7 +3,7 @@
     <input
       #inputField
       matInput
-      placeholder="Schools"
+      placeholder="Teams"
```
Comment (TheSlimvReal) [inline src/app/child-dev-project/schools/school-select/school-select.component.html:6]: I dont think this should be hardcoded. Maybe use the entity type?

File: src/app/child-dev-project/notes/model/note.ts [hunk 3]
```diff
@@ -72,6 +72,8 @@ export class Note extends Entity {
    */
   @DatabaseField() relatesTo: string;
 
+  @DatabaseField() schools: string[] = [];
```
Comment (TheSlimvReal) [inline src/app/child-dev-project/notes/model/note.ts:75]: What will this be used for?

**Review by TheSlimvReal — CHANGES_REQUESTED** [review]
Works very well, just a minor thing.

File: src/app/child-dev-project/schools/school-select/school-select.component.html (hunk 2 above)
Comment (sleidig) [reply src/app/child-dev-project/schools/school-select/school-select.component.html:6]: yes, waiting for PR #728 for a proper solution. I am reverting these - this was just a workaround to build a beta version for the next customer appointment
